plugins/tiddlywiki/freelinks/readme.tid

```diff
@@ -1,42 +1,85 @@
+
+!! 🔧 Advanced Configuration
+
+To customize which tiddlers trigger freelinking, modify the shadow tiddler [[$:/plugins/tiddlywiki/freelinks/macros/view]]. This tiddler is tagged `$:/tags/Macro/View`, ensuring it acts as a local macro in each view template. The default content is:
 
 ```
 <$set name="tv-freelinks" value={{$:/config/Freelinks/Enable}}/>
 ```
 
-That means that for each tiddler the variable tv-freelinks will be set to the tiddler $:/config/Freelinks/Enable, which is set to "yes" or "no" by the settings in control panel.
+This sets `tv-freelinks` to the value of `$:/config/Freelinks/Enable` (either "yes" or "no") for each tiddler, as defined in the control panel settings.
 
-Instead, we can use a filter expression to, say, only freelink within the tiddler with the title "HelloThere":
+For more specific control, you can use filter expressions. For example, to enable freelinking only within the tiddler titled "HelloThere":
 
 ```
 <$set name="tv-freelinks" value={{{ [<currentTiddler>match[HelloThere]then[yes]else[no]] }}}/>
 ```
 
-Or, we can make a filter that will only freelink within tiddlers with the tag "MyTag":
+To limit freelinking to tiddlers tagged "MyTag":
 
 ```
 <$set name="tv-freelinks" value={{{ [<currentTiddler>tag[MyTags]then[yes]else[no]] }}}/>
 ```
 
-Or we can combine both approaches:
+You can also combine conditions, such as freelinking only for "HelloThere" with the "MyTag" tag:
 
 ```
 <$set name="tv-freelinks" value={{{ [<currentTiddler>match[HelloThere]] ~[<currentTiddler>tag[MyTag]] +[then[yes]else[no]] }}}/>
 ```
+
+!! 🎨 Feature Highlights
+
+''🎯 Target Filter'': Use `$:/config/Freelinks/TargetFilter` to define which tiddlers get the auto-link treatment
+
+''🔤 Word Boundary Intelligence'' (Western languages):<br>
+  - ''Enabled'' (default): Links only complete words → @@__Yes__@@ eyes eYes (only "Yes" gets linked)<br>
+  - ''Disabled'': Links partial matches too → @@__Yes__@@ eyes e@@__Yes__@@ (both "Yes" instances get linked)
+
+''🧠 Smart Prioritization'': "Machine Learning Tutorial" gets linked before just "Machine" - context matters!
+
+''🚫 Self-Link Prevention'': No awkward self-references - tiddlers don't link to themselves
+
+!! ⚡ Under the Hood
+
+''Lightning-Fast Algorithm'': Powered by optimized Aho-Corasick with smart safeguards:<br>
+* ''Depth Protection'': Max 100 failure link transitions per character<br>
+* ''Scale Handling'': Processes up to 100K nodes (or 15× your pattern count)<br>
+* ''Result Limiting'': Cap of 10,000 matches per text block<br>
+* ''Memory Smart'': Uses TiddlyWiki's global cache for blazing performance
+
+''Bulletproof Reliability'':<br>
+* Invalid patterns? Skipped automatically<br>
+* Algorithm hiccups? Falls back gracefully to normal text<br>
+* Unicode chaos? Properly escaped and handled
+
+//Perfect for wikis with thousands of tiddlers - stress tested and ready for action!//
+
+!! 🙏 Credits & Thanks
```

Comment:
don't need to mention AI, they are our tool (I don't use the word "slave" because they don't feel), we can just take the work as ours.

And I think this is like thanking VSCode or Windows. Nowadays I believe most programmer use AI for years, so crediting this feels like you just start using AI, which is a shame...

Reply:
No offence, but I think it is time to 「看山不是山」